### ISSUE TITLE
How do I ignore public when regenerating all?

### ISSUE THREAD

**Issue by khuongduybui** [feathers-plus/generator-feathers-plus#116]
### Steps to reproduce
- I bundle a fully generated client-side application inside /public. The client-side application is huge by itself with >50,000 files (most of them come from the theme, not even my own client-side app, so I am very hesitant to cherry-pick the files).
- I add `public`, `public/*`, `public/**/*` to `freeze[]` in `feathers-gen-specs.json`.
- I run `feathers-plus generate all`.

### Expected behavior
I expect feathers-plus CLI to ignore my public directory and everything inside it

### Actual behavior
feathers-plus CLI tries to scan my public directory and errors out with

```
Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:514:3)
    at tryReadSync (fs.js:337:20)
    at readFileSync (fs.js:374:19)
    at extractFiles (/home/duybui/.config/yarn/global/node_modules/@feathers-plus/generator-feathers-plus/lib/code-fragments.js:76:25)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
```

**Comment by eddyystop** [comment]
feathers-gen-specs.json##options.freeze expects individual file names, not globs. So that's not appropriate for ignoring all of public/. I've added an issue regarding globs.

On the other hand the generator only writes these files in public/ **and only if they do not already exist**: `favicon.ico` and `index.html`.

I'll push a change to ignore public/ during the scan. You should be OK with that and the comments above. But backup your project the first time anyway. :)

**Comment by eddyystop** [comment]
Published in cli+ v0.7.66 which runs generator v0.6.64.

**Comment by khuongduybui** [comment]
I ran `npm i -g @feathers-plus/cli` and still got 0.7.66 and generator 0.6.64?

**Comment by eddyystop** [comment]
It looks like I had forgotten to publish f+/cli. Published cli+ v0.7.67 now using generator v0.6.65.
